### PR TITLE
Update Kafka release YAML

### PR DESCRIPTION
**Pull request by aliok**
Ran the `generate-kafka.sh` manually

### CODE REVIEW

File: openshift/release/knative-eventing-kafka-contrib-ci.yaml [hunk 1]
```diff
@@ -193,6 +193,11 @@ metadata:
   name: kafkabindings.bindings.knative.dev
 spec:
   group: bindings.knative.dev
+  preserveUnknownFields: false
```
Comment (matzew) [inline openshift/release/knative-eventing-kafka-contrib-ci.yaml:196]: Did you `regenerate` that ?

Comment (aliok) [reply openshift/release/knative-eventing-kafka-contrib-ci.yaml:196]: yeah I did. what's the issue?

Comment (matzew) [reply openshift/release/knative-eventing-kafka-contrib-ci.yaml:196]: No, just checking .

/lgtm
/approve

Comment (matzew) [reply openshift/release/knative-eventing-kafka-contrib-ci.yaml:196]: @aliok I think we wanna do the same for the 0.18 branch ?

Comment (aliok) [reply openshift/release/knative-eventing-kafka-contrib-ci.yaml:196]: yeah

**Pull request merged**